examples/common_vmseries/main.tf

```diff
@@ -14,6 +14,12 @@ locals {
   vmseries_password = coalesce(var.vmseries_password, try(random_password.this[0].result, null))
 }
 
+# Obtain Public IP address of code deployment machine
+
+data "http" "this" {
```

Comment:
maybe let's add here a condition, I do not see much sense in running this source if we do not use full bootstrapping with acl. So either check if a bootstrap storage is defined, or even better, check if you have bootstrap storage with network ACL defined

Reply:
Added a count condition to it to trigger only when the `bootstrap_storage` map variable is not empty and if the value of `storage_acl` is `true` :

```
data "http" "this" {
  count = length(var.bootstrap_storage) > 0 && contains([for v in values(var.bootstrap_storage) : v.storage_acl], true) ? 1 : 0
  url = "https://api.ipify.org"
}
```